app/assets/javascripts/initializers/initializeTouchDevice.js

```diff
@@ -10,23 +10,23 @@ function getClassList(className) {
 function blur(event, className) {
   setTimeout(() => {
     if (document.activeElement !== getById(className)) {
-      getClassList('navbar-menu-wrapper').remove('showing');
+      getClassList('top-bar__menu').remove('showing');
     }
   }, 10);
 }
 
 function removeShowingMenu() {
```

Comment:
For another PR, but this JS looks like it could just be hover states via CSS.

Reply:
definitely, I think we can give top-bar related some love generally. but as you said - for another PR :)